src/operator/random/sample_op.cc

```diff
@@ -268,5 +269,19 @@ Example::
                                                                        [ 6.,  4.]]
 )code" ADD_FILELINE);
 
+MXNET_OPERATOR_REGISTER_SAMPLE(_random_randint, SampleRandIntParam)
+.add_alias("random_randint")
```

Comment:
No need to add this alias if it's the same as the original name

Reply:
in that case should I also remove the redundant aliases for all the others too
random_uniform,random_normal,random_poisson,random_negative_binomial, etc

Reply:
as per discussion, keeping it, since there's a diff between `_random_randint` and `random_randint` Hence adding Alias makes sense in all the cases.